react-components/src/api/getData.ts

```diff
@@ -0,0 +1,18 @@
+import { Article } from '../helpers/types';
+
+export async function getData(): Promise<Article[]> {
+  try {
+    const response = await fetch(
+      'https://gnews.io/api/v4/search?q=example&lang=en&country=us&max=10&apikey=fb6ef00c29fbdfbd5cb3daf99a130b74',
+      {
+        method: 'GET',
+      }
+    );
+    const result = await response.json();
+    console.log(result);

```

Comment:
shouldn't be in a PR

Reply:
I can't find information about banned API in technical description. It is only used to get information for cards.

Reply:
Comment was about console.log()

Reply:
Ahh, sorry, that's understandable.